### PR TITLE
#52 Tuned SimpleCov profile

### PR DESCRIPTION
**Pull request by sergiy-koshoviy**
- Added .simplecov file with config
- Added new own profile

### CODE REVIEW

**Comment by sveredyuk** [comment]
Fixes #52

File: .simplecov [hunk 1]
```diff
@@ -0,0 +1,14 @@
+SimpleCov.profiles.define 'pivorak_cov' do
+  add_filter '/spec/'
+  add_filter '/config/'
+
```
Comment (denys-medynskyi) [inline .simplecov:4]: in the #52 there are requirements to exclude view helpers and channels

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]

File: .simplecov (hunk 1 above)
Comment (sergiy-koshoviy) [reply .simplecov:4]: @denysmedynskyi  Vova peredumav :)

Comment (sveredyuk) [reply .simplecov:4]: @denysmedynskyi helpers might be OK now as we **dryed** a lot of templates code there

**Review by sveredyuk — APPROVED** [review]

**Review by denys-medynskyi — APPROVED** [review]

**Pull request merged**